AEPEdgeConsent.podspec

```diff
@@ -15,8 +15,8 @@ Pod::Spec.new do |s|
   s.swift_version = '5.1'
 
   s.pod_target_xcconfig = { 'BUILD_LIBRARY_FOR_DISTRIBUTION' => 'YES' }
-  s.dependency 'AEPCore', '>= 3.1.0'
-  s.dependency 'AEPEdge', '>= 1.1.0'
+  s.dependency 'AEPCore', '>= 3.5.0'
+  s.dependency 'AEPEdge', '>= 1.4.0'
```

Comment:
This one is not a requirement, but are you bumping just for format consistency?

Reply:
Yes, for the format consistency.